tests/ssl-opt.sh

```diff
@@ -10197,6 +10189,39 @@ run_test    "TLS 1.3: HelloRetryRequest check, ciphersuite TLS_AES_256_GCM_SHA38
             -c "Protocol is TLSv1.3" \
             -c "HTTP/1.0 200 OK"
 
+requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_3
+requires_config_enabled MBEDTLS_SSL_TLS1_3_COMPATIBILITY_MODE
```

Comment:
```suggestion
requires_config_disabled MBEDTLS_SSL_TLS1_3_COMPATIBILITY_MODE
```
server side hasn't support it